data-prepper-plugins/dynamodb-source/src/main/java/org/opensearch/dataprepper/plugins/source/dynamodb/converter/MetadataKeyAttributes.java

```diff
@@ -12,9 +12,11 @@ public class MetadataKeyAttributes {
 
     static final String SORT_KEY_METADATA_ATTRIBUTE = "sort_key";
 
-    static final String EVENT_TIMESTAMP_METADATA_ATTRIBUTE = "ts";
+    static final String EVENT_TIMESTAMP_METADATA_ATTRIBUTE = "ddb_ts";
 
     static final String EVENT_NAME_BULK_ACTION_METADATA_ATTRIBUTE = "op";
 
+    static final String DDB_STREAM_EVENT_NAME_METADATA_ATTRIBUTE = "ddb_op";
```

Comment:
Similar to my question above, why use `op`? I'm not even quite sure - is this an "operation?"